mypy/subtypes.py

```diff
@@ -616,6 +617,30 @@ def g(x: int) -> int: ...
 
         In this case, the first call will succeed and the second will fail: f is a
         valid stand-in for g but not vice-versa.
+
+    allow_potential_compatibility:
+        By default, this function returns True if and only if left is
+        definitely compatible with right.
```

Comment:
Instead of "definitely" maybe add an explanation in terms of subtyping? This shouldn't be mathematically precise (I could guess why you are mostly using "compatibility" instead of "subtyping", but for historical reasons in mypy code compatibility is checked by `is_subtype`, while subtyping by `is_proper_subtype`). Maybe say something like _each_ call that succeeds for right, will guaranteed to succeed for left callable.